BambooTracker/midi/midi.hpp

```diff
@@ -24,12 +24,13 @@
  */
 
 #pragma once
-#include "RtMidi/RtMidi.hpp"
+
 #include <string>
 #include <vector>
 #include <memory>
 #include <mutex>
 #include <cstdint>
+#include "RtMidi.hpp"
```

Comment:
The original headers were simply `*.h`, we renamed our in-tree versions for the pedantic compiler settings. This fails to compile for me because it's looking for `RtMidi.hpp` instead of `RtMidi.h`.